data/xp-pen-ack05-remote.tablet

```diff
@@ -0,0 +1,41 @@
+# XP-Pen
+# ACK05 Remote
+#
+# Reports as Hanvon Ugee Shortcut Remote, for recordings
+# etc. see https://gitlab.freedesktop.org/libevdev/udev-hid-bpf/-/issues/32
+# and
+# sysinfo.PUwwSY53x2.tar.gz
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/415
+#
+#  ┌────────┐────────────────────────┐
+# /  ┌────┐  \   ┌───┐┌───┐┌───┐┌───┐│
+# │  │ A  │  │   │ B ││ C ││ D ││   ││
+# │  └────┘  │   └───┘└───┘└───┘│ H ││
+# \_________/    ┌───┐┌───┐┌───┐│   ││
+#  │             │ E ││ F ││ G ││   ││
+#  │             └───┘└───┘└───┘└───┘│
+#  │             ┌───┐┌────────┐┌───┐│
+#  │             │ I ││    J   ││ K ││
+#  │             └───┘└────────┘└───┘│
+#  └─────────────────────────────────┘
+
+[Device]
+Name=Hanvon Ugee Shortcut Remote
```

Comment:
```suggestion
Name=XP-Pen ACK05 Remote
```
The name is the one to display to the user, that the device reports itself as something different on the kernel doesn't matter, better to not confuse the users here.

If there is an actual ugee remote then we can change this.